src/gui/dialogs/DialogRadioButton.cpp

```diff
@@ -111,9 +111,19 @@ void RadioButton::draw_n_btns(size_t btn_count) const {
     if (!texts)
         return;
 
+    static_assert(sizeof(btn_count) < 5, "Too many RadioButtons to draw.");
+
     Rect16 splits[4]; //fix size, dont want to use template
     Rect16 spaces[3];
-    rect.VerticalSplit(splits, spaces, btn_count, GuiDefaults::ButtonSpacing);
+    uint8_t ratio[4];
```

Comment:
```
const size_t maxButtons = 4;
Rect14 splits[maxButtons];
Rect14 spaces[maxButtons-1];
uint8_t ratio[maxButtons];
```
Or define `static const uint_32 MAX_DIALOG_BUTTONS = 4;` somewhere in the code (guidefaults.hpp?) and use it from there.